README.md

```diff
@@ -6,13 +6,19 @@ Based on [benchmarks published by NVidia for their RAPIDS](https://render.github
 # Run benchmarks
 
 ## GPU case
-- launch `p3.2xlarge` gpu instance on AWS cloud (this instance has a NVIDIA V100 GPU)
+- launch `p3.2xlarge` gpu instance on AWS cloud with `Ubuntu Server 18.04 LTS (HVM), SSD Volume Type` image (this instance has a NVIDIA V100 GPU)
 - login in it
 - follow [the steps](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html#Cluster_GPUs_Manual_Install_Driver) to install the necessary driver for V100 GPU (using 396.82 version)
-- `git clone https://github.com/vnlitvin/mortgage-benchmarks.git`
-- `cd mortgage-benchmarks`
-- `chmod +x test_mortgage_gpu.sh && ./test_mortgage_gpu.sh`
-- `bash`
-- `conda activate cudf_dev`
-- `pip install xgboost`
-- `time python Mortgage_GPU.py  test_mortgage_gpu/dataset/ 1`
+- `git clone https://github.com/vnlitvin/mortgage-benchmarks.git && cd mortgage-benchmarks`
+- `git checkout amyachev`
```

Comment:
is this needed after the merge? seems not

Reply:
fixed